package-lock.json

```diff
@@ -16315,7 +16315,7 @@
         },
         "has-flag": {
           "version": "3.0.0",
-          "resolved": "https://a0us.jfrog.io/a0us/api/npm/npm/has-flag/-/has-flag-3.0.0.tgz",
```

Comment:
One other thing - I see a lot of these. Do you expect this? Why does it happen?

Reply:
Not exactly sure, but it's probably due to a setting change on our registry.

https://www.jfrog.com/confluence/display/RTF/Npm+Registry#npmRegistry-AutomaticallyRewritingExternalDependencies

Reply:
It looks like this changed originally with @ubenzer 's PR on 1/20. @ubenzer Wdyt? It seems like the npm registry is the correct one.

Reply:
Verbally replied, but adding here for completeness as well. Yeah my bad. I didn't notice it. They should point to npm. 🤦‍♂